src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,36 @@
+.clock {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+
+  &__circle {
+    position: relative;
+    @include circle($size);
+    background-color: $color-circle-second;
+    border-radius: 50%;
+    align-self: center
+  }
+
+  &__minute {
+    position: absolute;
+    align-self: center;
+    height: $height-minute;
+    width: $width-line;
```

Comment:
move common styles for hands to mixin ([DRY](https://www.techtarget.com/whatis/definition/DRY-principle))